modules/ocf_irc/files/znc.service

```diff
@@ -0,0 +1,11 @@
+[Service]
+ExecStart=/usr/bin/znc --foreground --datadir /opt/znc
+User=znc
+Group=znc
+Restart=always
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=znc
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
[The ZNC documentation](https://wiki.znc.in/FAQ#How_to_reload_ZNC_config_without_restart.3F) makes it sound like doing this is highly discouraged.

It might make sense to do a SIGUSR1 followed by SIGHUP, but I don't really know to what end. It seems like it writes to disk any time you save something in the web interface, so I don't know when the running config would differ from the written one. Perhaps if you connected to IRC as the admin user and did some commands, but I don't think we want to encourage that anyways.